drivers/windows/R/paths.R

```diff
@@ -66,9 +66,13 @@ print.windows_path <- function(x, ...) {
 }
 
 
-remote_path <- function(x, shares) {
+remote_path <- function(x, shares, run_on_linux = FALSE) {
   x <- prepare_path(x, shares)
-  file.path(x$drive_remote, x$rel)
+  if (run_on_linux) {
+    path_on_linux(x)
```

Comment:
this name I think is insufficiently clear as to what sort of changes are going on here, and in particular I think that this is really "path on the dide network as seen from a linux machine", as opposed to some sort of windows/linux slash normalisation that one might expect

Reply:
Renamed.